examples/fluids/include/petsc_ops.h

```diff
@@ -25,12 +25,14 @@ PetscErrorCode OperatorApplyContextCreate(DM dm_x, DM dm_y, Ceed ceed, CeedOpera
 PetscErrorCode OperatorApplyContextDestroy(OperatorApplyContext op_apply_ctx);
```

Comment:
Also, thinking about a different name for the struct, as at this point it's starting to look a lot more like a `CeedOperator`, but for Petsc Vecs rather than just a context. Maybe something like `PetscCeedOperator`? Or `FEMOperator` since it can implements the global finite element operator (where `CeedOperator` only implements the local version)?

Reply:
Functional changes now and a better name maybe in the future if there is consensus?

Reply:
Yeah, sounds fine by me.